listenstore/listenstore/listenstore.py

```diff
@@ -25,6 +25,9 @@
 
 REDIS_USER_TIMESTAMPS = "user.%s.timestamps" # substitute user_name
 USER_CACHE_TIME = 3600 # in seconds. 1 hour
+LISTENCOUNT_CACHE_TIME_LOW = 15 * 60 # in seconds. 15 minutes
```

Comment:
We might want to move these defines into the class which uses them. Makes the code a little cleaner/easy to read.